admin/class-ed-solr-admin.php

```diff
@@ -180,18 +180,9 @@ public function index_all_blogs_in_solr() {
 			$posts = get_posts( -1 );
 
 			foreach ( $posts as $post ) {
-				$document = $update->createDocument();
```

Comment:
[x] Line 152: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 153: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 154: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 155: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 156: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 158: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 158: Opening statement of multi-line function call not indented correctly; expected 8 spaces but found 9
 [x] Line 159: Multi-line function call not indented correctly; expected 13 spaces but found 12
 [x] Line 168: Multi-line function call not indented correctly; expected 13 spaces but found 12
 [x] Line 169: Multi-line function call not indented correctly; expected 9 spaces but found 8
 [x] Line 183: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 184: Tabs must be used to indent lines; spaces are not allowed
 [x] Line 195: Tabs must be used to indent lines; spaces are not allowed
 [ ] Line 257: The $domain arg must be a single string literal, not "$this->ed_solr".
 [ ] Line 258: The $domain arg must be a single string literal, not "$this->ed_solr".

Reply:
Hopefully the tabs have been fixed.

And 257 & 258 should work as they are run through __() translate function which return a literal.

Reply:
ok